test/autocomplete/EmojiProvider-test.js

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2022 Ryan Browne <code@commonlawfeature.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import EmojiProvider from '../../src/autocomplete/EmojiProvider';
+
+const EMOJI_SHORTNAMES = [
+    ':+1',
+    ':heart',
+    ':grinning',
+    ':hand',
+    ':man',
+    ':man:_beard',
+    ':sweat',
+    ':monkey',
+    ':boat',
+    ':mailbox',
+    ':cop',
+    ':bow',
+    ':kiss',
+    ':golf',
+];
+
+const TOO_SHORT_EMOJI_SHORTNAME = [
+    [':o', '⭕'],
+];
+
+describe('EmojiProvider', function() {
+    it.each(EMOJI_SHORTNAMES)('Returns consistent results after final colon %s', async function(emojiShortcode) {
+        const ep = new EmojiProvider('test-room');
+        const range = { "beginning": true, "start": 0, "end": 3 };
+        const completionsBeforeColon = await ep.getCompletions(emojiShortcode, range);
+        const completionsAfterColon = await ep.getCompletions(emojiShortcode + ':', range);
+
+        const firstCompletionWithoutColon = completionsBeforeColon.map(x => x.completion)[0];
+        const firstCompletionWithColon = completionsAfterColon.map(x => x.completion)[0];
+
+        expect(firstCompletionWithoutColon).toEqual(firstCompletionWithColon);
+    });
+
+    it.each(TOO_SHORT_EMOJI_SHORTNAME)("Returns correct results after final colon %s", async ({ emojiShortcode, expectedEmoji }) => {
```

Comment:
This line is not correct for the style, but I am not certain how to fix it. I tried matching the style in `matrix-js-sdk/spec/unit/crypto/CrossSigningInfo.spec.js` but that does not seem to be correct for the style in this repo. So, could someone point me toward the correct style to be matching?

Reply:
The line is just too long, perhaps something like this could work:

```suggestion
    it.each(TOO_SHORT_EMOJI_SHORTNAME)("Returns correct results after final colon %s", async (
        { emojiShortcode, expectedEmoji }
    ) => {
```

Reply:
Thanks! Got that resolved.